CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+##2015-04-15 - Supported Release 1.0.0
+###Summary
+Add ability to reboot linux distro's as well as Windows.
+
+###Features
+* Round up to nearest minutes for linux reboot timeout
```

Comment:
This may not be necessary, as there was never an actual release with Linux support.  That being said, there's not currently a note about Linux timeouts needing to be whole minutes in the readme.  Will submit a fix ASAP.